src/command_modules/azure-cli-profile/HISTORY.rst

```diff
@@ -2,6 +2,8 @@
 
 Release History
 ===============
+2.1.3
```

Comment:
@yugangw-msft please check this version number in particular to ensure that it should be a patch bump and doesn't call for a minor bump.